scripts/mypy

```diff
@@ -1,6 +1,8 @@
-#!/usr/bin/env python3
-"""Mypy type checker command line tool."""
+#!/usr/bin/env bash
```

Comment:
I don't feel strongly about what you're doing with Windows, but I feel the changes to this file are not an improvement. How do you even know the relationship between the python binary and this script?